Sources/SentryCrash/Recording/SentryCrashC.c

```diff
@@ -171,6 +171,12 @@ sentrycrash_install(const char *appName, const char *const installPath)
     return monitors;
 }
 
+void
+sentrycrash_uninstall(void)
+{
+    sentrycrashccd_close();
```

Comment:
I looked at the implementation of `sentrycrashccd_close()` but don't really understand what or how the operations there are "closing".

Reply:
It stops the thread running `monitorCachedData` to update `updateThreadList` in `SentryCrashCachedData.c`

Reply:
Ah thanks, I didn't follow through to `monitorCachedData` 👍🏻